cameraview/src/main/java/com/otaliastudios/cameraview/CameraController.java

```diff
@@ -140,8 +138,8 @@ void stopImmediately() {
             LOG.i("Stop immediately. Stopped. State is:", ss());
         } catch (Exception e) {
             // Do nothing.
-            LOG.i("Stop immediately. Exception while stopping.", e);
             mState = STATE_STOPPED;
+            mCameraCallbacks.onError("Stop immediately. Exception while stopping.", e);
```

Comment:
This can be ignored as it was, `stopImmediately` is called on `camera.destroy()` and we expect that it *might* fail because in some cases it can run in parallel with `camera.stop()`.

It's safe to say here that the camera was released, we did our best, also we don't want to throw exceptions on activity destroy.